### PR TITLE
Dynamic keypairs 2

### PR DESCRIPTION
**Pull request by alexvas**
## Changes
*    Removed broken CA/client certificates with client private key from repository.
*    Fixed keypairs generating script, added two another scripts to install generated CA script in Ubuntu
*    Updated documentation

## Verification
```
yarn test:integration
```
works for me now.

### CODE REVIEW

**Comment by johanbrandhorst** [comment]
Stop opening new pull requests. You can keep pushing to the same branch.

**Comment by alexvas** [comment]
ok. Let's see if I solved travis build right this time.

File: go/grpcwebproxy/main.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ var (
 
 	flagAllowAllOrigins = pflag.Bool("allow_all_origins", false, "allow requests from any origin.")
 	flagAllowedOrigins  = pflag.StringSlice("allowed_origins", nil, "comma-separated list of origin URLs which are allowed to make cross-origin requests.")
+	flagAllowedHeaders  = pflag.StringSlice("allowed_headers", []string{}, "comma-separated list of headers which are allowed to propagate to the gRPC backend.")
 
 	runHttpServer = pflag.Bool("run_http_server", true, "whether to run HTTP server")
 	runTlsServer  = pflag.Bool("run_tls_server", true, "whether to run TLS server")
```
Comment (alexvas) [inline go/grpcwebproxy/main.go]: This is from my previous PR that was already merged. I am not sure what would be right thing to do. Should I rebase my commits into ws-request-headers (the branch already merged) and make PR from there?

Comment (johanbrandhorst) [reply go/grpcwebproxy/main.go]: Yes, rebase on current master please.

Comment (alexvas) [reply go/grpcwebproxy/main.go]: Well, yet another PR: #589

**Pull request closed without merging**